extensions/amp-analytics/amp-analytics.md

```diff
@@ -72,6 +72,11 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
     <amp-analytics config="https://example.com/analytics.config.json"></amp-analytics>
     ```
 
+  - `data-consent-notification-id` Optionally make the request be blocked
```

Comment:
Optional attribute. If provided will stop processing the analytics request until a `amp-user-notification` with the given HTML-id was confirmed by the user.

Please also link the tag name to the actual docs.